app/posts/find-teacher-training/2021-08-16-standardised-entry-requirement-descriptions.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Standardising entry requirement content
+description: Why we have made the descriptions of entry requirements more consistent
+date: 2021-08-16
+---
+
+Prior to the 2022 to 2023 cycle, providers were asked to describe the academic qualification entry requirements for each course as a single free text field.
+
+As there is a minimum initial teacher training entry requirement across all courses, this led to the standard entry requirements being described in different ways by different providers.
+
+For instance:
+
+> GCSE Maths, English and Science at Grade C or higher or equivalent
+
+> GCSEs: At least a Grade C or Grade 4 at GCSE (or equivalent) in English Language, Mathematics and Science is essential for all applicants.
+
+> GCSE in English (or approved equivalent) at grade C/Level 4 or above GCSE in Mathematics (or approved equivalent) at grade C/Level 4 or above GCSE in Science (or approved equivalent) at grade C/Level 4 or above
+
+These minor inconsistencies may be confusing for candidates, and the requirements weren’t always written in the easiest to understand way.
```

Comment:
Change to: 'These small differences may be confusing for candidates. And the way providers had written requirements could be difficult to understand.'